angular.json

```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```

Comment:
Do you know what this is?

Reply:
The analytics is to share data with the Angular Team. The change occurred after running the upgrade process.